extensions/pyRevitTools.extension/pyRevit.tab/Analysis.panel/Tools.stack/Analyse.pulldown/Get In Place Family.pushbutton/script.py

```diff
@@ -1,26 +1,35 @@
 """Get a total count and types of In Place Family in the current document.
+Also isolate them in a new 3Dview called InPlace Families.
 
-Copyright (c) 2019 Jean-Marc Couffin
+Copyright (c) 2021 Jean-Marc Couffin
 https://github.com/jmcouffin
 """
 #pylint: disable=import-error,invalid-name,broad-except,superfluous-parens
-from pyrevit import revit, DB
-from pyrevit import script
+import clr
```

Comment:
this is where I did not know if there was a better way to create the iCollection, could not find anything in pyRevit modules